cmd/commands/cd.go

```diff
@@ -0,0 +1,58 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/twpayne/go-shell"
+	"github.com/urfave/cli/v2"
+	"github.com/version-fox/vfox/internal"
+	"os"
+	"os/exec"
+)
+
+var Cd = &cli.Command{
+	Name:  "cd",
+	Usage: "Launch a shell in the VFOX_HOME or SDK directory",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:    "plugin",
+			Aliases: []string{"p"},
+			Usage:   "Launch a shell in the plugin directory",
+		},
+	},
+	Action: cdCmd,
+}
+
+func cdCmd(ctx *cli.Context) error {
+	userShell, _ := shell.CurrentUserShell()
```

Comment:
There is no need to introduce third-party libraries, `vfox` has its own simple implementation.

https://github.com/version-fox/vfox/blob/e48be9318d96807212b7c95308425e8364e6c34b/internal/sdk.go#L298

Reply:
This method requires get the PID of the shell process.

Reply:
You can use this method to get parent PID, which is the shell process PID.

https://github.com/version-fox/vfox/blob/e48be9318d96807212b7c95308425e8364e6c34b/internal/env/flag.go#L34-L40